tests/test_meshtastic_utils.py

```
@@ -2444,6 +2444,32 @@ def test_seed_connect_time_skew_allows_one_prestart_bootstrap_on_reconnect(self)
+        mu._relay_connection_started_monotonic_secs = 1_000.0
+        mu.RELAY_START_TIME = 190_000.0
+        mu._relay_startup_drain_deadline_monotonic_secs = None
+        mu._relay_reconnect_prestart_bootstrap_deadline_monotonic_secs = None
+
+        with (
+            patch("mmrelay.meshtastic_utils.time.time", return_value=now_wall),
+            patch("mmrelay.meshtastic_utils.time.monotonic", return_value=now_mono),
+        ):
+            result = mu._seed_connect_time_skew(rx_time)
+
+        self.assertTrue(result)
```

Ruff (0.15.9)

[warning] Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)